src/components/test/operate.test.js

```diff
@@ -0,0 +1,9 @@
+import Operate from '../../logic/operate';
+
+// Test calculation of numbers
```

Comment:
- [ ] Kindly add more tests that cover all functionalities & scenarios of the operate.js and calculate.js modules.